docs/guides/platforms/rpi/running-on-raspbian-lite-via-drm.md

```diff
@@ -156,7 +156,7 @@ public static int Main(string[] args)
     if (args.Contains("--drm"))
     {
         SilenceConsole();
-        return builder.StartLinuxDrm(args);
+        return builder.StartLinuxDrm(args: args, card: "/dev/dri/card1", scaling: 1.0);
```

Comment:
```suggestion
        // By default, Avalonia will try to detect output card automatically.
        // But you can specify one, for example "/dev/dri/card1".
        return builder.StartLinuxDrm(args: args, card: null, scaling: 1.0);
```

Reply:
It's important to mention, as otherwise people will complain when card1 is not available for some reason.
See https://github.com/AvaloniaUI/Avalonia/blob/0b72cdb512187c8a4eded47e864e1194111e621f/src/Linux/Avalonia.LinuxFramebuffer/Output/DrmBindings.cs#L144-L164